src/Compilers/CSharp/Test/Symbol/Symbols/SymbolEqualityTests.cs

```diff
@@ -911,6 +920,179 @@ public static void M(A<T> t)
                 );
         }
 
+        [Fact]
+        public void TestImplementationAttributes_Inlining()
```

Comment:
>TestImplementationAttributes_Inlining [](start = 20, length = 37)

It looks like we are lacking tests for VB.

Reply:
@AlekseyTs I'll add them.